packages/types/test/unit/ExecutionAddress.test.ts

```diff
@@ -0,0 +1,61 @@
+import {describe, it, expect} from "vitest";
+import {toChecksumAddress} from "../../src/utils/ExecutionAddress.js";
+
+describe("toChecksumAddress", () => {
+  it("should fail with invalid addresses", () => {
+    expect(() => toChecksumAddress("0x1234")).toThrowError("Invalid address");
+  });
+
+  it("should format address as ERC55", () => {
+    type TestCase = {
+      address: string;
+      checksumedAddress: string;
```

Comment:
its either

```suggestion
      checksummedAddress: string;
```

or just (preferable)

```suggestion
      checksumAddress: string;
```